oak_dice/src/evidence.rs

```diff
@@ -215,7 +202,7 @@ impl ApplicationKeys {
 static_assertions::assert_eq_size!([u8; 2048], ApplicationKeys);
 
 /// ECDSA private keys that can be used for an application for signing or encryption.
-#[derive(AsBytes, FromZeroes, FromBytes)]
+#[derive(AsBytes, FromZeroes, FromBytes, Zeroize, ZeroizeOnDrop)]
```

Comment:
Wow, it has zeroize on drop already, nice
Will use it for the private key in the crypto as well